modules/store/src/Cache/Context/StoreCacheContext.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\commerce_store\Cache\Context\StoreCacheContext.
+ */
+
+namespace Drupal\commerce_store\Cache\Context;
+
+use Drupal\commerce_store\StoreContextInterface;
+use Drupal\Core\Cache\Context\CacheContextInterface;
+use Drupal\Core\Cache\CacheableMetadata;
+
+/**
+ * Defines the StoreCacheContext service, for "per store" caching.
+ *
+ * Cache context ID: 'store'.
+ */
+class StoreCacheContext implements CacheContextInterface {
+
+  /**
+   * The store context.
+   *
+   * @var \Drupal\commerce_store\StoreContextInterface
+   */
+  protected $storeContext;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getLabel() {
+    return t('Store');
```

Comment:
Let's use the ``StringTranslationTrait`` and use ``$this->t(`` here instead if that's possible.